CONDUCT.md

```diff
@@ -0,0 +1,37 @@
+## Open Source Code of Conduct
+
+### Summary
+
+This code of conduct demonstrates expectations for participation in ssg_kmeansr open source and how to report unacceptable behaviours. We are aiming to provide a welcoming and encouraging experience for all parties. People violating this Code of Conduct may be excluded from the open source practice. 
+
+### What we encourage 
+
+As contributors and administrators of this open source, we encourage all parties to contribute by, and not limited to, raising issues, initiating function requests, updating documentations and pulling requests. 
```

Comment:
..., raising issues, initiating ~~function~~ **feature** requests, ... and ~~pulling~~ submitting pull requests

Reply:
@guanchenz thank you so much for your comment! I got these from the resource I read and I just tried to make some changes in the wording...I will change function to feature, but I thinking pulling request should be okay~